### PR TITLE
Jh/performance preloader

### PR DESCRIPTION
**Pull request by Hiro-Nakamura**
## Release Notes

- improve site loading through parallel loads
 

NOTE: this update requires updates to `api_sails`, `appbuilder`, `tenant_manager` and `user_manager` projects to also use the `#jh/performancePreloader` branches.

### CODE REVIEW

File: init/Preloader.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      };
+      // Fire the loading
+      head.appendChild(script);
+   });
+}
+
+async function Preload() {
+   let allScripts = [];
+
+   allScripts.push(ScriptLoad("/config/site"));
```
Comment (nh758) [inline init/Preloader.js:27]: Is this file needed? it only adds `/config/site` script which is also added by `/config/preloader`?

File: init/initDefinitions.js [hunk 2]
```diff
@@ -1,8 +1,11 @@
 import { nanoid } from "nanoid";
 export default {
-   init: async (BS) => {
+   init: async (/* BS */) => {
       // BS {Bootstrap}
       // The initial Bootstrap object found in "./Bootstrap.js"
+      /* 
+       * Definitions now loaded in preloader.
```
Comment (nh758) [inline init/initDefinitions.js:7]: we can delete this file, right?

File: webpack/index.ejs [hunk 3]
```diff
@@ -9,7 +9,7 @@
       <title>AppBuilder</title>
       <!-- https://www.filamentgroup.com/lab/load-css-simpler/#the-code -->
       <link href="/__getCookie" rel="stylesheet" media="print" onload="this.media='all'" />
-      <script type='text/javascript' src='/assets/dependencies/sails.io.js' reconnection="true" defer></script>
+      <script type='text/javascript' src='/config/preloader'></script>
```
Comment (nh758) [inline webpack/index.ejs:12]: Can you add `defer` here? This will allow the browser to render our preload UI and keep our first contentful paint score high.

**Review by nh758 — CHANGES_REQUESTED** [review]

File: init/initDefinitions.js (hunk 2 above)
Comment (Hiro-Nakamura) [reply init/initDefinitions.js:7]: yeah, I've cleaned that up now.

File: webpack/index.ejs (hunk 3 above)
Comment (Hiro-Nakamura) [reply webpack/index.ejs:12]: ok, made some changes to enable this.  

turns out, this is a good update and shaves some more time off the load.

**Review by nh758 — APPROVED** [review]
Looks good
